hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -227,6 +227,9 @@ if (fqtn != null && master.isInitialized()) {
     <li class="active">
```

Comment:
While testing it, when I had first loaded table.jsp page, none of these tabs get selected, which made both "base stats" and "localities" info get rendered, becoming a bit polluted and confusing. Attached [screenshot](https://issues.apache.org/jira/secure/attachment/13005481/Screen%20Shot%202020-06-11%20at%2013.01.56.png) to the jira, I guess we should make "base stats" tab selected by default (like making the link in main page point to "table.jsp?name=test#baseStats")?

Reply:
It is a fixed issue: HBASE-24496, do you testing with newest code of master?

Reply:
Seems i missed some info that both get rendered, let me test local.

Reply:
Fixed the both tabs rendered issue, and the other issue that "base stats" not selected by default had been fixed in HBASE-24496 few days ago. 
Thanks.